doc/overview.md

```diff
@@ -10,7 +10,7 @@ The testbed is intended as a playground. Further services and integration will b
 and experiences from the productive installations will be included here in the future. It will become more production-like over
 time. However, at no point does it claim to represent a production exactly.
 
-The section [How to contribute](/contributing.md) describes how to contribute to this testbed. Open Source Software lives from
+The section [How to contribute](./contributing.md) describes how to contribute to this testbed. Open Source Software lives from
```

Comment:
this has to be `./contribute.md` now after renaming.

Reply:
That´s right. Thank you for the hint. It´s done :)